PedalCore/Models/Pedal.swift

```diff
@@ -7,8 +7,63 @@
 
 import Foundation
 
-struct Pedal: Hashable {
+struct Pedal: Identifiable {
+    var id: UUID = UUID()
     var name: String
     var brand: String
-    var knobs: [String: Int]
+    var knobs: [Knob]
+    
+}
+
+
+extension Pedal {
```

Comment:
Please move this extension to a new file, so we can exclude it from the official builds.

Reply:
It makes totally sense!